entries/n/netcorecloud.com.json

```
@@ -0,0 +1,13 @@
+{
+  "Netcore Cloud": {
+    "domain": "netcorecloud.com",
```

```suggestion
    "domain": "netcorecloud.com",
    "additional-domains": [
      "pepipost.com"
    ],
```

The `pepipost.com` domain still seems to be in use despite the renaming.